ci_build_images/msan.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# this is to create images with MSAN for BB workers
+WORKDIR /tmp/msan
+
+COPY build-msan2.sh .
+
+RUN curl -sL https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor -o /usr/share/keyrings/llvm-snapshot.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" > /etc/apt/sources.list.d/llvm-toolchain.list \
+    && echo "deb-src [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/llvm-snapshot.gpg] \
+    http://apt.llvm.org/jammy/ llvm-toolchain-jammy-15 main" >> /etc/apt/sources.list.d/llvm-toolchain.list \
+    && apt-get update \
+    && echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90forceyes \
+    && apt-get -y install --no-install-recommends dpkg-dev devscripts clang-15 \
+    && apt-get source libc++-15-dev \
+    && mv llvm-toolchain-15-15* llvm-toolchain \
+    && mkdir llvm-toolchain/build \
+    && cmake \
+        -S llvm-toolchain/runtimes \
+        -B llvm-toolchain/build \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_C_COMPILER=clang-15 \
+        -DCMAKE_CXX_COMPILER=clang++-15 \
+        -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi" \
+        -DLLVM_USE_SANITIZER=Memory \
+    && make -C llvm-toolchain/build install -j10 \
+    && apt-get source gnutls28 nettle libidn2 libgmp10 libcrack2 \
```

Comment:
please use one package per line and sort them alphabeticaly, see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

Reply:
And you can change names, see bellow:
```console
Picking 'gmp' as source package instead of 'libgmp10'
Picking 'cracklib2' as source package instead of 'libcrack2'
```

Reply:
Use:
`make -C llvm-toolchain/build install -j$(nproc)` to be consistent with the rest.

Reply:
Everything is corrected as suggested.